src/main/window.js

```diff
@@ -170,6 +154,7 @@ class AppWindow {
 
     // set renderer arguments
     const { codeFontFamily, codeFontSize, theme } = userPreference.getAll()
+    // wow, this can be accessesed in renderer process.
```

Comment:
Can you please explain?

Reply:
> Can you please explain?

You accessesed the stylePrefs in App.vue in renderer process.

> `const style = win.stylePrefs || DEFAULT_STYLE`